woostarter/patterns/about-background-image-offset-text.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Title: Background image, offset text and link
+ * Slug: woostarter/background-image-offset-text-link
+ * Categories: text, about, media
+ * Keywords: about, text, icons, cover, story
+ * Description: A section with a background image and some offset text.
+ *
+
+?>
+<!-- wp:cover {"url":"https://kiri.mystagingwebsite.com/wp-content/uploads/2024/11/home-pattern-cover-image-scaled-1.jpg","id":217,"alt":"A black and white abstract composition of curved paper with light and shadow.","dimRatio":30,"overlayColor":"contrast","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","sizeSlug":"full","metadata":{"name":"Background image, offset text and link"},"align":"full","style":{"spacing":{"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
```

Comment:
This `src` attribute needs to be updated to point to the themes assets directory (`<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/home-pattern-cover-image-scaled-1.jpg`) before merging.